src/main/java/com/github/pedrovgs/androidwifiadb/view/AndroidDevices.java

```diff
@@ -0,0 +1,170 @@
+package com.github.pedrovgs.androidwifiadb.view;
+
+import com.github.pedrovgs.androidwifiadb.adb.ADB;
+import com.github.pedrovgs.androidwifiadb.adb.ADBParser;
+import com.github.pedrovgs.androidwifiadb.adb.AndroidWiFiADB;
+import com.github.pedrovgs.androidwifiadb.adb.CommandLine;
+import com.github.pedrovgs.androidwifiadb.model.Device;
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationGroup;
+import com.intellij.notification.NotificationType;
+import com.intellij.notification.Notifications;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.openapi.wm.ToolWindowFactory;
+import com.intellij.ui.content.Content;
+import com.intellij.ui.content.ContentFactory;
+
+import javax.swing.JPanel;
+
+/**
+ * Created by vgaidarji on 10/23/15.
+ */
+public class AndroidDevices implements ToolWindowFactory, View, DeviceAction {
```

Comment:
If this is a window inside the IDE we should rename this class to ``AndroidWiFiADBWindow`` as we are already doing with ``AndroidWiFiADBAction``.